include/config/config_movement.h

```diff
@@ -113,6 +113,11 @@
  */
 #define DONT_LEDGE_GRAB_STEEP_SLOPES
 
+/**
+ * Buffers an A input if you jump off a slope during the landing lag
+ */
+//#define SLOPE_BUFFER
```

Comment:
Can be enabled by default

Reply:
disagree, movement changes shouldnt be enabled by default

Reply:
it is not a movement change per say - it is just buffering the A press so jumps do not get "eaten". The only thing it does is give more leaveaway for pressing A to get from the ground so it should be acceptable. No vanilla or custom level will be broken by it.

Reply:
i think movement changes shouldnt be on by default, but stuff like this where only the most extreme players would care about it (same frame ground pound falls in this camp too)

Reply:
Yeah, if we're going as far as to revert EASIER_LONG_JUMPS, stuff like this isn't getting through either

Reply:
Formatting nitpick:
```suggestion
// #define SLOPE_BUFFER
```